pylarky/eval/http_evaluator.py

```diff
@@ -1,12 +1,21 @@
+import re
+import json
+
 from pylarky.eval.evaluator import Evaluator
-from pylarky.request import HttpRequest
+from pylarky.model.request import HttpRequest
 
 
 class HttpEvaluator(Evaluator):
 
-    def __init__(self, script, http_request: HttpRequest):
-        super(HttpEvaluator, self).__init__(script, http_request.to_starlark())
+    def __init__(self, script):
```

Comment:
BTW why do you need __init__ here at all?

Reply:
For possible future usage of using same script for multiple payloads